src/components/ExpandedBankTransactionRow/ExpandedBankTransactionRow.tsx

```diff
@@ -644,6 +663,7 @@ export const ExpandedBankTransactionRow = forwardRef<SaveHandle, Props>(
                         href={url}
                         target='_blank'
                         rel='noopener noreferrer'
+                        // onClick={openReceiptInNewTab(url, index)}
```

Comment:
Kept this commented out since it does not work for PDFs and would technically be a regression but feel free to try locally. The actual solution here will be using the fileType field on the S3 URL ("application/pdf" or "image/png") and varying behavior accordingly

Reply:
Noting here that I am not actually doing so in this PR because we should get https://github.com/Layer-Fi/layer-react/pull/358 merged first